.github/workflows/python-app.yml

```diff
@@ -10,28 +10,44 @@ on:
     branches: [ master ]
 
 jobs:
-  build:
+  build-all:
+    name: ${{ matrix.os }}, python ${{ matrix.python-version }}
+    runs-on: ${{ matrix.os }}
+    
+    strategy:
+      matrix:
+        os: ["ubuntu-latest", "macos-latest", "windows-latest"]
+        python-version: ["3.7", "3.8", "3.9"]
 
-    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l {0}
 
     steps:
     - uses: actions/checkout@v2
-    - name: Set up Python 3.9
-      uses: actions/setup-python@v2
+    
+    - name: Setup environment
+      uses: conda-incubator/setup-miniconda@v2
       with:
-        python-version: 3.9
-    - name: Install dependencies
+        auto-update-conda: true
+        python-version: ${{ matrix.python-version }}
+        mamba-version: "*"
+        channels: conda-forge
+        channel-priority: strict
+        environment-file: environment.yml
+        activate-environment: geoutils
+
+    - name: Install development dependencies and package
       run: |
-        $CONDA/bin/conda install -y -c conda-forge mamba
-        $CONDA/bin/mamba env update --file environment.yml --name base
-        $CONDA/bin/pip install -r dev-requirements.txt
-        $CONDA/bin/pip install .
+        mamba install sphinx numpydoc sphinx_rtd_theme sphinx-autodoc-typehints sphinxcontrib-programoutput flake8 pytest matplotlib pylint
+        pip install -e . --no-dependencies
+
     - name: Lint with flake8
       run: |
         # stop the build if there are Python syntax errors or undefined names
-        $CONDA/bin/flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
-        # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
```

Comment:
The comment why you set to 127 was useful, why remove it?

Reply:
I didn't remove it; it's just called with `flake8` instead of `$CONDA/bin/flake8` to work better with Windows!